docs/modules/general/pages/bootstrapping.adoc

```diff
@@ -0,0 +1,567 @@
+            deps: [HttpClient, AppStateService]
+        },
+        missingTranslationHandler: {
+            provide: MissingTranslationHandler,
+            useClass: PortalMissingTranslationHandler
+        }
+        }),
+        BrowserModule,
+        AngularAuthModule,
+        BrowserAnimationsModule,
+    ],
+    exports: [],
+    providers: [
+        {
+            provide: Configuration,
+            useFactory: apiConfigProvider,
+            deps: [ConfigurationService, AppStateService]
+        },
+        {
+            provide: APP_INITIALIZER,
+            useFactory: initializeRouter,
+            multi: true,
+            deps: [Router, AppStateService]
+        },
+        provideHttpClient(withInterceptorsFromDi())
+    ]
+})
+export class ExampleModule implements DoBootstrap {
+    constructor(private readonly injector: Injector) {}
+
+    ngDoBootstrap(): void {
+        createAppEntrypoint(AppEntrypointComponent, 'example-component', this.injector)
+    }
+}
+....
+
+The **module.ts** file is prepares the module for integration with OneCX platform.
+
+This example showcases the recommended approach of defining modules (Angular based) using Webcomponent method. Here is a list of important features of this example:
+
+Module imports:: 
+* CommonModule, BrowserModule and BrowserAnimationsModule Angular modules used for adding functionality to the module.
+* PortalCoreModule is defined to allow usage of OneCX components and services.
+* TranslateModule is defined to allow translations using translation keys within the module.
+// TODO: Add link to auth docs
+* AngularAuthModule is defined to use OneCX authorization mechanisms.
+* RouterModule is defined for routing to feature modules within the exposed module.
+
+Entrypoint component::
+// TODO: Add link to createAppEntrypoint
+AppEntrypoint is a standard Angular component, which has a `<router-outlet>` element in its template. The `createAppEntrypoint` registers AppEntrypointComponent in the Custom Elements Registry, so anytime '<example-component>' is rendered, AppEntrypointComponent should be instantiated.
++
+The third parameter, being the module's Injector, is very important. This injector will be used by the instances of AppEntrypointComponent rendered using Web Components technology, meaning that each instance will have everything related to the module already set up. That also means the `<router-outlet>` will be using routes defined for the module.
++
+// TODO: Add link to createAppEntrypoint
+// TODO: Link to events topic
+[[module-rotuer-connection]]The `createAppEntrypoint` method is also responsible for connecting the module's router to Shell's router. Every time the URL of the browser changes, Shell is going to publish a new message, via EventsTopic, with information about the new URL. The `createAppEntrypoint` method subscribes to the EventsTopic and updates the router state accordingly to the received information.
```

Comment:
... connecting the module's router to **the** Shell's router. Every time the URL of the browser changes, **the** Shell is going to publish a new message ...